letsdrill.rb

```diff
@@ -1,24 +1,34 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  letter_grade = ""
+  if integer >= 90
+    letter_grade = "A"
+  elsif integer >= 80
+    letter_grade = "B"
+  elsif integer >= 70
+    letter_grade = "C"
+  elsif integer >= 60
+    letter_grade = "D"
+  else
+    letter_grade = "F"
+  end
+  return letter_grade
```

Comment:
The `return` keyword is optional in ruby, and the standard is to omit it.  If you did that here, can you see how you could also get rid of the `letter_grade` variable?